exploration/number-selection.md

```diff
@@ -431,9 +514,9 @@ With the proposed design, this message would much more naturally be written as:
 
 ```
 .input {$count :number minimumFractionDigits=1}
-.match {$count}
-0 {{You have no apples}}
-1 {{You have exactly one apple}}
+.match $count
+0.0 {{You have no apples}}
+1.0 {{You have exactly one apple}}
```

Comment:
```suggestion
0 {{You have no apples}}
1 {{You have exactly one apple}}
```

Reply:
This depends on whether the fraction digits apply or not. It doesn't matter, because the context is proposing a separate `:plural` selector.

Reply:
There are traps if we don't compare numeric.

Most locales will format currencies (by default) with 2 decimals, but some with 3, and some with none.

So if the source starts with 
```
0.00 {{This is free today!}}
```
then in some locales this will never match.

Because they would format to `0` or `0.000`. And that's the default. No attributes specified by the developers.

Worse, there are regions using the same language formatting the currency differently, because the local currency has sub-units or not.

So (for example) the Arabic translation would have to have keys for both `0.00` and `0.000`, so that exact match works for various countries.